presto-spi/src/main/java/com/facebook/presto/spi/security/AccessDeniedException.java

```diff
@@ -128,6 +128,16 @@ public static void denyCreateView(String viewName, String extraInfo)
         throw new AccessDeniedException(format("Cannot create view %s%s", viewName, formatExtraInfo(extraInfo)));
     }
 
+    public static void denyCreateViewWithSelect(String viewName)
```

Comment:
Since the argument of the method `denyCreateViewWithSelect` can be either a view name or a table name, perhaps rename `viewName` as `viewOrTableName`.